conftest.py

```diff
@@ -169,10 +169,12 @@ def client(project):
 def dataset(client):
     """Create a dataset."""
     with client.with_dataset(name='dataset') as dataset:
-        dataset.author = {
-            'name': 'me',
-            'email': 'me@example.com',
-        }
+        dataset.creator = [{
```

Comment:
With the deleted test below, we only have 1 more remaining which is the import from git (which is not really working properly). We should probably address this at some point. Do we have a ticket for that?

I would add those additional checks into `test_git_repo_import` and assert them as well.